### DISCUSSION TITLE
Read custom settings from appsettings.json

### DISCUSSION THREAD

**Discussion by MaxGasta** [OrchardCMS/OrchardCore#7124] [Q&A] [answered]
Hello,
how can I read some customs setting stored in appsettings.json ?

eg. some Build Info  to show in the footer

Thanks for help

**Comment by hishamco** [comment, marked as answer]
You can use Configuration APIs or Shells in OC, inject `IShellConfiguration` then use `GetSection()` like normal configuration APIs
